fiftyone/core/aggregations.py

```diff
@@ -167,6 +170,20 @@ def default_result(self):
         """
         raise NotImplementedError("subclasses must implement default_result()")
 
+    def _must_transform_output_field_value(self):
+        """Whether the output value of this aggregation must be transformed
+        with parse_result(). Note that this must be called after to_mongo() for
+        the field to be set.
+
+        Returns:
+            True/False
+        """
+        if hasattr(self, "_field"):
```

Comment:
realized that I can only check this after to_mongo() has been called... so thinking of safer alternative to pass this info without leaking into collections.py